nntrainer/layers/layer_context.h

```diff
@@ -134,13 +134,6 @@ class InitLayerContext {
     return weights_spec;
   }
 
-  /**
-   * @brief Get the current tensors spec
-   *
-   * @return The current tensors spec
-   */
-  const std::vector<TensorSpec> &getTensorsSpec() const { return tensors_spec; }
```

Comment:
please done delete this. This is needed by the graph :).

Reply:
Restored